Log-Sources/Tausight/src/parsers/tausight-json.yaml

```
@@ -0,0 +1,80 @@
+  "1311459764055923618", "17367035759339725179", "16614933329223078581", "6229110698169467111",
+  "9544895561993643442", "13312185421629450693", "10734140536465371835", "17110226208560029986",
+  "3233025372120568883", "16070293335899322576", "6146200383242126333", "1990852215035401417",
+  "16211170557513602992", "5632628727331703358", "1687631264888284218"], "executable":
+  {"fileAttributes": ["FILE_ATTRIBUTE_ARCHIVE"], "creationTime": "2019-12-07T09:09:44.320193500Z",
+  "lastAccessTime": "2023-11-15T22:19:04.247864900Z", "companyName": "Microsoft Corporation",
+  "description": "Location Notification", "fileVersion": "10.0.19041.1 (WinBuild.160101.0800)",
+  "fileSize": "71168", "productName": "LocationNotification", "productVersion": "10.0.19041.1"},
+  "signature": {}}, "session": {"authenticationPkg": "Kerberos", "caption": " ", "description":
+  " ", "logonId": 553834, "logonSessionType": "LS_INTERACTIVE"}}, "domain": "BH",
+  "domainAndHost": "BH\\MXL04543WX", "messageId": "4e42dc0dfbdc2dfd4f27aef3ed8d14150211c4088a537fd5748aa3493a7a9976",
+  "eventType": "application_started"}'
+fieldsToBeRemovedBeforeParsing: []
+$schema: https://schemas.humio.com/parser/v0.2.0
+script: "// tausight-json parser for ingesting tausight sensor telemetry\n// Copyright\
+  \ tausight 2023\n\n// Drop events that are not used\ncase {\n @rawstring=\"*moduleInventory*\"\
+  \ or\n @rawstring=\"*driverInventory*\" or\n @rawstring=\"*servicesInventory*\"\
+  \ or\n @rawstring=\"*processInventory*\"\n| dropEvent();\n * }\n\n// Parse as json\
+  \ and exclude unused or arrays with excessive fields that can't be handled\n// default\
+  \ error handling is to drop the first 100 elements and then continue rather than\
+  \ stopping at the 100th.\n| parseJson(exclude=[applicationStarted.mainProcess.modules,\
+  \ tauoneStarted.drivers, tauoneStarted.productList])\n\n// drop events that are\
+  \ prior to 2.4 agent release\n//|version = /(?<versionMajor>^\\d+)\\.(?<versionMinor>\\\
+  d+)\\./i\n//| case{versionMajor<2 or (versionMajor=2 and versionMinor<3)|dropEvent();*}\n\
+  \n// Tag endpoint state events\n| case {\n    devicePoweredUp or devicePoweredDown\
+  \ or\n    tauoneStarted or tauoneShutdown or\n    encryptionChanged or deviceAttached\n\
+  \    | eventTag := \"endpoint\";\n\n    fileInspected\n    | eventTag := \"fileInspected\"\
+  ;\n\n    email\n    | eventTag := \"email\";\n\n    cloudStorage or fileDeleted\
+  \ or\n    fileCopied or fileReplaced or fileMoved or\n    clipboardActivity or fileIoActivity\n\
+  //    M365 or\n    | eventTag := \"fileActivity\";\n\n    applicationStarted\n \
+  \   | eventTag := \"application\";\n\n    sessionStarted or sessionTerminated or\
+  \ sessionEvent\n    | eventTag := \"session\";\n\n    heartbeat\n    | eventTag\
+  \ := \"heartbeat\" ;\n    *\n    |eventTag := \"others\"\n}\n\n| parseTimestamp(field=\"\
+  timestamp\", as=@timestamp, timezone=UTC) "
+tagFields:
+- eventTag
```

Only `Vendor` field should be tagged
```suggestion
- Vendor
```